src/gvars.c

```diff
@@ -684,10 +684,12 @@ Obj FuncIDENTS_GVAR (
     /*QQ extern Obj          NameGVars;   */
     Obj                 copy;
     UInt                i;
+    Obj                 strcopy;
 
     copy = NEW_PLIST( T_PLIST+IMMUTABLE, LEN_PLIST(NameGVars) );
     for ( i = 1;  i <= LEN_PLIST(NameGVars);  i++ ) {
-        SET_ELM_PLIST( copy, i, ELM_PLIST( NameGVars, i ) );
```

Comment:
I would suggest adding a comment here that explains why we make the copy. Else somebody might unwittingly "improve" this code again.